assets/config/assets.onj

```diff
@@ -203,7 +211,7 @@ textures: [
     },
     {
         name: "nav_bar_background",
-        file: "textures/nav_bar/background.png"
+        file: "textures/statusbar/background_old.png"
```

Comment:
because some screen still use the old one, and the new background has corners on the edge (ik there are more and maybe better names, but since it will be deleted soon anyway, i didn't care